frontend/finance-tracker/src/validations/form-schema.ts

```
@@ -31,3 +31,50 @@ export const registrationFormSchema = z
+    currency: true,
+});
+
+export const transferFundFormSchema = z.object({
+    amount: z.coerce
+        .string()
+        .min(1, {
+            message: "Amount is required",
+        })
+        .refine(
+            (value) => {
+                const amount = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)